.cloud-build/tests/select_notebooks_test.py

```diff
@@ -0,0 +1,33 @@
+import sys
+sys.path.append("..")
```

Comment:
Oh don't do this. You can set PYTHONPATH in the envvar of the test runner instead.

Reply:
removed